src/content-scripts/pageTransition.event.content.js

```diff
@@ -128,8 +128,8 @@
          * property from this module's click content script and background script. The cached page
          * visits are from the same tab as this page or, if the page is opened in a new tab, the cached
          * page visits are from the opener tab.
-         * @param {boolean} message.isOpenedTab - Whether the page is loading in a new tab that was
-         * opened by another tab.
```

Comment:
I think we should keep `isOpenedTab`. There are some situations where a boolean is convenient, especially since -1 isn't falsy in JavaScript.